bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js

```diff
@@ -63,6 +63,12 @@ class FeatureDataPluginUIHandler extends StateHandler {
     }
 
     setActiveTab (layerId) {
+        // when clicking on a tab directly, layerId will be a number.
+        // When clicked in the "show more" menu of the tablist, it will be a String and no features are found.
+        // so we gotta make a conversion for a numeric layerid just in case. :(
+        if (Oskari.util.isNumber(layerId)) {
+            layerId = +layerId;
```

Comment:
Good commenting above 👍 Maybe use parseInt() instead to make it easier to understand?